server/routes/users.js

```diff
@@ -1,8 +1,12 @@
 const express = require('express');
 const mongoose = require('mongoose');
+const jsonwebtoken = require('jsonwebtoken');
 
 const router = express.Router();
 
+const localStrategyAuth = require('../utils/localStrategy');
+const jwtStrategy = require('../utils/jwtStrategy');
+
 router.get('/', (req, res) => {
```

Comment:
@gualopezb let's delete that kitty model and also the route that uses it, i think we don't need it anymore

Reply:
@juanprq you're right. I just deleted the kitty model.

Reply:
lol hello kitty